style.css

```diff
@@ -1,7 +1,78 @@
 html {
-  font-size: 10px;
+  font-size: 12px;
 }
-title {
-  background-color: #3E3C3C;
-  color: blue;
+
+@media only screen and (max-width: 500px) {
+  .wrapper {
+    max-width: 1000px;
+    margin: 0 auto;
+    padding: 50px;
+    background-color: #1c1a19;
+    display: flex;
+    flex-direction: column;
```

Comment:
- I guess the reason why the styles are not applied is that you have used the wrong syntax here. Please use `@media screen and (max-width: "your desired screen width")`
**[OPTIONAL]** - I also advise that you use a `max-width` of 960px. This will ensure your site displays properly on all small screen sizes.

Reply:
Hi, thanks for the feedback. I did it on purpose. The idea is that the CSS format will apply only to small devices. In my case 500px, I think we should set up the project for a mobile application at this stage. You may see the changes when you reduce the browser size. If I do the changes you suggest, the application would be for a desktop app size. Should I do the changes even though?
Best regards 
Edahí